applications/app/services/ApplicationsInteractiveRendering.scala

```diff
@@ -18,13 +19,24 @@ object USElection2020AmpPage extends RenderingTier
   1. Implement the support for it in DCR, or
   2. Implement support for it directly in the [applications] app, using the AMP page already present in CAPI.
 
-  The former wold have taken too long so we went for the latter.
+  The former would have taken too long so we went for the latter.
+
+  ----------------
+  Author: Pascal
+  Date: 21st April 2021
+
+  We are now moving towards supporting interactives in DCR 🙂
  */
 
 object ApplicationsInteractiveRendering {
   def getRenderingTier(path: String)(implicit request: RequestHeader): RenderingTier = {
     val isSpecialElection = ApplicationsUSElection2020AmpPages.pathIsSpecialHanding(path)
     val isAmp = request.host.contains("amp")
-    if (isSpecialElection && isAmp) USElection2020AmpPage else Regular
+    val forceDCR = request.forceDCR
+    (isSpecialElection, isAmp, forceDCR) match {
+      case (true, true, _)  => USElection2020AmpPage
+      case (_, false, true) => DotcomRendering
```

Comment:
Should this be `(_, _, true)`? Or are we starting with non-amp only for some reason?

Reply:
@jfsoul 
I refactored, but the answer to your question is yes, I am thinking starting with non amp for the moment.